### DISCUSSION TITLE
Programmatically disable a grid. (making readonly)

### DISCUSSION THREAD

**Discussion by b-giavotto** [6pac/SlickGrid#989] [Q&A] [answered]
I use a slickgrid with RowSelectionModel. based on user choice (for example a selection of a checkbox) i need to completely disable the ability to select a cell or row either by mouse or keyboard.
Is there a such kind? How can i get this?

**Comment by 6pac** [comment]
@b-giavotto sorry for delay. You can just use no selection model at all. Or I think there are options to disable selection.

**Comment by b-giavotto** [comment]
Thx, i've found there is an "editable" option in slick grid options.

**Reply by ghiscoding** [reply, marked as answer]
change it dynamically `grid.setOptions({ editable: false });`
